js_modules/dagster-ui/packages/ui-core/src/search/BuildAssetSearchResults.tsx

```diff
@@ -22,7 +22,7 @@ type CountPerTag = {
   assetCount: number;
 };
 
-type CountPerGroupName = {
+export type CountPerGroupName = {
```

Comment:
Not related?

Reply:
this is also used in https://github.com/dagster-io/internal/pull/9048